packages/fxa-react/configs/rescripts.js

```diff
@@ -4,14 +4,10 @@
 
 const { resolve } = require('path');
 
-const additionalJSImports = {
-  'fxa-react': resolve(__dirname, '../'),
-  'fxa-shared': resolve(__dirname, '../../fxa-shared'),
-};
-
 const permitAdditionalJSImports = (config) => {
-  const importPaths = Object.values(additionalJSImports);
-
+  // We're just gonna call all of fxa fair game ;)
+  const allFxa = resolve(__dirname, '../../')
+  const importPaths = [allFxa, resolve(__dirname, '../../../node_modules')]
```

Comment:
May need some additional the [tsconfigs](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/tsconfig.json#L17-L24) that previously resolved the specific paths. Or, maybe it can be moved to `tsconfig.browser.json` since it's fxa-wide now?

Reply:
I'm not sure I follow. Each of the packages that use rescripts has it's own tsconfig and should add it's own references, which btw don't matter to react-scripts, only tsc.